torchaudio/extension/__init__.py

```diff
@@ -1,7 +1,9 @@
 from .extension import (
     _init_extension,
+    _init_transducer_extension,
 )
 
 _init_extension()
+_init_transducer_extension()
```

Comment:
I think this initialization should happen in `torchaudio/prototype/__init__.py` so that the non-prototype part of the code is fully-unaware of prototype.